src/src/DataTypes/SensorVType.h

```diff
@@ -33,7 +33,7 @@ enum class Sensor_VType : uint8_t {
   SENSOR_TYPE_DOUBLE_SINGLE   =   70,
   SENSOR_TYPE_DOUBLE_DUAL     =   71,
 #endif
-
+  SENSOR_TYPE_EASYFETCH       =   254,
```

Comment:
I don't think you should need this new Sensor_VType 🤔

Reply:
Why not? This is supposed to be my "switch" for enabling an options field and the json output.

Reply:
Just leaving the args empty should be enough?

Reply:
> Just leaving the args empty should be enough?

you are talking about having the easyfetch argument field no matter what is selected  as the "Output Data Type"?
But if there will be ever a formula field in the dummy plugin it gets a bit crowded?

Reply:
Having all the decimals and stats options already makes it crowded, maybe we should use a 2-rows layout once we add the formula option. Will think some more about this.